client/src/components/EventCard/ProgrammingLanguages.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box } from "@mui/material";
+import NodeJS from "../../assets/ProgrammingLanguages/nodejs.png";
+import ExtraLanguages from "./ExtraLanguages";
+
+export const thresholdOfLanguages = 4;
+
+export const languages = [
+    "NodeJS",
+]as const;
+
+export type Language = typeof languages[number];
+
+export const images: Record<Language, string> = {
+    "NodeJS": NodeJS,
+}
+
+const ProgrammingLanguage = ({language}:{language: Language}) => {
+    return (
+        <Box
+            component="img"
+            sx={{
+                display: "flex",
+                alignItems: "flex-end",
+                justifyContent: "flex-end",
+                width: "5%",
+                height: "100%",
+                backgroundColor: "text.disabled",
+                padding: "1%",
+                borderRadius: "10%",
+                marginRight: "1%",
+            }}
+            alt={language}
+            src={images[language]}
+        >
+        </Box>
+    );
+}
+
+
+const ProgrammingLanguages = ({languages}:{languages: Language[]}) => {
+    
+    const languagesSize = languages.length;
+
+    const showDropdownButton = languagesSize > thresholdOfLanguages ? 1 : 0;
```

Comment:
0 or 1 is unneeded here languageSize > thresholdOfLanguages is already a boolean expresison